content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,380 @@
+
+To learn more about PAS, see the concept Staking, [Penalty-Absorbing Stake (PAS)](/concepts/staking/#penalty-absorbing-stake-pas).
+
+## Rules
+
+### Accountability rules
+
+Accountability rules are applied to detect faults in the three Tendermint consensus round phases *propose*, *prevote*, *precommit*. Rules are detailed in the table below.
+
+{{% alert title="Info" color="info"%}}
+In the table:
+
+- Rule ID: is the unique identifier for the Rule defined in the AFD rule engine. ID prefixes correspond to Tendermint consensus phases:
+  - `P`: *propose*
+  - `PV`: *prevote*
+  - `C`: *precommit*
+{{% /alert %}}
+
+| Rule ID | Description |
+| --| --|
+| `PN` - `PO` | Propose step related consensus rule infraction. | 
+| `PVN` - `PVO` - `PVO12` - `PVO3` | Prevote step related consensus rule infraction. |
+| `C` - `C1` | Precommit step related consensus rule infraction. |
+| `InvalidProposal` | Proposer has proposed a block that fails blockchain validation |
+| `InvalidProposer` | Proposer of a block proposal is not the committee's elected Proposer. |
+| `Equivocation` | Validator has sent conflicting messages during a consensus round: a committee member sending multiple prevotes/precommits for different values, a  proposer has broadcast conflicting block proposals to different committee members. |
+| `InvalidRoundStep` | Consensus round message contains invalid round number or step |
+| `WrongValidRound` | Consensus round message contains wrong valid round number |
+| `GarbageMessage` | Consensus round message was signed by sender, but it cannot be decoded |
+
+### Rule severity
+
+Rules are given a severity rating according to the risk that failure to adhere to the rule brings to block chain finality and integrity.
+
+{{% alert title="Note" %}}
+In this release all offences are treated as `Mid` severity.
+
+Future releases will introduce an offence scale range. 
+{{% /alert %}}
+
+<!--
+| Severity | Description |
+| --| --|
+| `Minor` | _Not currently implemented_ |
+| `Low` | Invalid message: invalid proposal, invalid voting message |
+| `Mid` | Equivocation message: (a) proposer has sent conflicting block proposal messages; (b) committee member has sent conflicting prevote or precommit messages. Inconsistent message: (a) proposer sends new proposal having already pre-committed for an earlier value in an earlier round; (b) proposer sends an old value that the proposer has never locked on. |
+| `High` | _Not currently implemented_ |
+| `Critical` | _Not currently implemented_ |
+-->
+
+## Events
+
+### Event handling
+
+Accountability _events_ are submitted to the system by validator's submitting proofs of behaviour from consensus messaging on-chain. See [`handleEvent()`](/reference/api/aut/op-prot/#handleevent-accountability-contract) for a description of event handling logic.
```

Comment:
Latest wording:

> Accountability _events_ are submitted on-chain by validator's submitting proofs of behavior from consensus messaging. See [`handleEvent()`](/reference/api/aut/op-prot/#handleevent-accountability-contract) for a description of event handling logic.

I still don't know what this means.

Part of the problem was the use of the possessive: "validator's" doesn't make sense in this context. Did you mean the plural, "validators"?

Another issue is "from consensus messaging" - what does this mean?

Reply:
Hmm, I'm trying to say that it's gotten from the Tendermint gossip between validators during consensus round phases.  Perhaps this is clearer:

>Proofs are derived from data exchanged between validators as they propose, prevote, and precommit blocks during Tendermint consensus rounds.  The proofs are then submitted on-chain to the Accountability Contract.

Reply:
No, it should align exactly with the description in https://docs.autonity.org/concepts/consensus/pos/#consensus-round-and-internal-state:

>>In each consensus instance a consensus round of three voting steps is executed as a sequential exchange of cryptographically signed (i.e. ‘sealed’) messages broadcast between committee members:

Reply:
So this might be better:

>Proofs are derived from cryptographically signed messages broadcast between committee members during Tendermint consensus rounds as validators propose, prevote, and precommit blocks (see the concept description [Consensus round and internal state](/concepts/consensus/pos/#consensus-round-and-internal-state)). The proofs are then submitted on-chain to the Accountability Contract.

Reply:
Done in commit b248094eb28f21c1d4cfd82be33257d6484b34be!